src/GeniusYield/Test/Privnet/Setup.hs

```diff
@@ -157,9 +201,8 @@ withPrivnet testnetOpts setupUser = do
                         . poolRuntime
                         $ head poolNodes
                 , runtimeNetworkInfo = GYNetworkInfo
-                    { gyNetworkEra        = era
-                        -- TODO: Conway support.
-                    , gyNetworkEpochSlots = fromIntegral $ cardanoEpochLength testnetOpts
+                    { -- TODO: Conway support.
```

Comment:
I'm assuming this Conway support TODO can be removed?

Reply:
And any others that may exist in the codebase.

Reply:
I noticed that todo, but didn't want to spend time seeing what this code snippet is about. I have removed it now.